app/src/main/java/org/oppia/android/app/translation/AppLanguageResourceHandler.kt

```diff
@@ -123,6 +123,10 @@ class AppLanguageResourceHandler @Inject constructor(
   /** See [OppiaLocale.DisplayLocale.formatDouble] for specific behavior. */
   fun formatDouble(value: Double): String = getDisplayLocale().formatDouble(value)
 
+  /** See [OppiaLocale.DisplayLocale.toHumanReadableString] for specific behavior. */
+  fun toHumanReadableString(number: Int): String =
+    getDisplayLocale().toHumanReadableString(number).toString()
```

Comment:
Why ``.toString()``? If it's because of the nullability, I think we could go ahead and remove the nullable property part from OppiaLocale since it never seems to return null (I'm not actually sure why that's there right now).

Reply:
The nullable property has been removed from the methods in the files of origin.